lonestar/analytics/cpu/cdlp/cdlp_cli.cpp

```diff
@@ -84,6 +80,14 @@ main(int argc, char** argv) {
   katana::StatTimer totalTime("TimerTotal");
   totalTime.start();
 
+  if (symmetricGraph) {
+    KATANA_LOG_WARN(
+        "This application requires a symmetric graph input;"
+        " Using the -symmetricGraph flag "
+        " indicates that the input is a symmetric graph and can be used as it "
+        "is.");
+  }
```

Comment:
this needs to be removed now

Reply:
It is a warning that is shown once the --symmetricGraph flag is used indicating that katana assumes that no modification needs to be done to the input graph. 

I have followed the same warning message as in Louvain and Kcore merged PRs.

Reply:
ok, my mistake